engine-tests/src/tests/repro.rs

```diff
@@ -74,7 +74,7 @@ fn repro_FRcorNv() {
         #[cfg(not(feature = "ext-connector"))]
         near_gas_used: 168,
         #[cfg(feature = "ext-connector")]
-        near_gas_used: 167,
+        near_gas_used: 168,
```

Comment:
Funny that one gas cost went up while another went down.

Reply:
I guess we can get back to only one field because now the value is the same.